client/jira.go

```diff
@@ -140,9 +140,9 @@ func GetIssueInfo(issueId string) (domain.Issue, error) {
 	return issue, nil
 }
 
-func CreateIssue(project, issueType, title string) (domain.Issue, error) {
+func CreateIssue(project, issueType, summary string) (domain.Issue, error) {
 	var issue domain.Issue
-	issueRequest := newCreateIssueRequest(project, issueType, title)
+	issueRequest := newCreateIssueRequest(project, issueType, summary)
```

Comment:
This is personal preference, but I'm not a fan of having a separate function to create our structure here 🤔 what I suggest we do instead is simply create the structure in the scope of this function:

```go
issue := domain.Issue{}
issueRequest := IssueRequest {
   ProjectRequest{projectKey}
   ...
   ..
 }
```
doing this and deleting the `newCreateIssueRequest` in my mind makes it easier to follow what's going on, this file is a bit convoluted with code as it is so I think it'd be worth it to do this refactor. wdyt?